tv_apps/dashboard/style/dashboard.css

```diff
@@ -199,6 +213,57 @@ iframe {
 /* Intro animation */
 
 /*
+ * Animation for background.
+ */
+
+/* XXX: Animating the backgroun image with over 2k resolution causes performance problem. */
+/*      Might have to remove this animation later. */
+.active .background {
+  animation: shrink 1s cubic-bezier(0.0, 0.0, 0.0, 1.0) 1.2s;
+}
+
+@keyframes shrink {
+  from { transform: scale(1.2); }
+  to { transform: scale(1); }
```

Comment:
Per discussion we can try 3d tranform.
1. Add `transform-style: preserve-3d; perspective: 25rem;` in `body`
2. transform from `translateZ(10rem)` to `translateZ(0)`

Values are not precisely measured so they may need to be adjusted.